src/calcium/ca.jl

```diff
@@ -0,0 +1,1447 @@
+###############################################################################
+#
+#   ca.jl : Calcium field elements
+#
+###############################################################################
+
+export ca, CalciumField, infinity, unsigned_infinity, undefined, unknown,
+       const_pi, const_euler, const_i, complex_normal_form, csgn, isnumber,
+       isimaginary, isalgebraic, isspecial, isinf, isuinf, is_signed_inf,
+       isunknown, isundefined, pow, gamma, erf, erfi, erfc
+
+###############################################################################
+#
+#   Data type and parent methods
+#
+###############################################################################
+
+parent(a::ca) = a.parent
+
+parent_type(::Type{ca}) = CalciumField
+
+elem_type(::Type{CalciumField}) = ca
+
+base_ring(a::CalciumField) = Union{}   #  ?
```

Comment:
I think these are correct. No need for the ?

Reply:
Yeah, I copied this from another module and I was just not sure about the meaning.